app/assets/javascripts/3dbio_viewer/src/domain/entities/Pdb.ts

```diff
@@ -40,6 +40,9 @@ export interface PdbLigand {
     name: string;
     inChI: string; //IUPACInChIkey
     imageDataResource?: LigandImageData;
+    smiles?: string;
+    formula?: string;
+    pubchemId?: string;
```

Comment:
The typical question: do we really want these props to be optional or to be `prop: Maybe<string>` so we don't forget about them when building?